### DISCUSSION TITLE
Gdpr and Directus

### DISCUSSION THREAD

**Discussion by biapar** [directus/directus#7828] [Archive]
Do Directus respect Gdpr rules? If yes, How?
Thanks.

**Comment by biapar** [comment]
?

**Comment by rijkvanzanten** [comment]
Directus itself doesn't store any pii in any centralized servers, or use any third party services internally that might do so. At the end of the day GDPR compliance is all around what _you_ do with Directus, as the platform itself doesn't store any (personal) data ootb.

**Comment by arungahlot-woolies** [comment]
> ?

Exactly My Friend... Pure "Rent Seeking" .... these are vampires and hold your email accounts to ransom.
